src/main/java/racingcar/Application.java

```diff
@@ -5,6 +5,6 @@
 public class Application {
     public static void main(String[] args) {
         RacingManager manager = new RacingManager();
-        manager.racingController().start();
+        manager.createRacingController().start();
```

Comment:
각각 어떤 느낌인지 생각해보는 것도 좋을 것 같아요 😄 

```suggestion
        RacingController racingController = manager.createRacingController()
        racingController.start();
```

```suggestion
        manager.createRacingController()
               .start();
```